packages/shared/lib/services/sync/config/deploy.service.ts

```diff
@@ -45,13 +45,32 @@ const nameOfType = 'sync/action';
 
 type FlowWithVersion = Omit<IncomingFlowConfig, 'fileBody'>;
 
+interface OrchestratorInterface {
```

Comment:
you could export this interface from the orchestrator and install as devDevp

Reply:
good idea. I like this

Reply:
I don't even need to do that. There is already a OrchestratorClient interface and I can use that to instantiate a mock orchestrator with a noop OrchestratorClient in the test. [2862bca](https://github.com/NangoHQ/nango/pull/2298/commits/2862bca38b2df1618de4432fa57847e42f131ddb)